units/github_status_api.go

```diff
@@ -81,6 +85,17 @@ func NewGithubStatusUpdateJobForExternalPatch(patchID string) amboy.Job {
 	return job
 }
 
+func NewGithubStatusUpdateJobForPushToCommitQueue(owner, repo, ref string, prNumber int) amboy.Job {
+	job := makeGithubStatusUpdateJob()
+	job.UpdateType = githubUpdateTypePushToCommitQueue
+	job.Owner = owner
+	job.Repo = repo
+	job.Ref = ref
+
+	job.SetID(fmt.Sprintf("%s:%s-%d-%s", githubStatusUpdateJobName, job.UpdateType, prNumber, time.Now().String()))
```

Comment:
Maybe the job name could reflect the owner/repo/ref instead of the PR number (since multiple projects could have overlapping PR number)?

Reply:
If we go this route, I think owner, repo, and number might be better, since it's easier to go from those to a URL.

Reply:
I changed the format to use owner, repo, and PR number.